frontend/js/src/utils/APIService.ts

```diff
@@ -1003,6 +1003,26 @@ export default class APIService {
     return response.json();
   };
 
+  updatePinRecordingBlurbContent = async (
+    userToken: string,
+    rowId: number,
+    blurbContent: string
+  ): Promise<{ status: string }> => {
```

Comment:
See comment below about the return value of the API endpoint.

Reply:
Hi, could you please elaborate the issue? I don't see a comment related to return value of API endpoint

Reply:
My bad! I don't know what I did but that was probably left in-between some comments I modified